### DISCUSSION TITLE
How to use filters in point cloud layers?

### DISCUSSION THREAD

**Discussion by ethanruiyu** [visgl/deck.gl#5630] [Q&A]
e.g: filter layers with z greater than 2m

**Comment by padawannn** [comment]
For example, you could have height property and use the data filter extension https://deck.gl/docs/api-reference/extensions/data-filter-extension

**Reply by ethanruiyu** [reply]
Thank you for your reply， but point cloud data is a FloatArray( [x,y,z,x,y,z...]), do I need to traverse it to get the height attribute, so I think it will reduce performance
